build/yaml/pipelines/telephony.yml

```diff
@@ -7,14 +7,11 @@ pool:
 
 extends:
   template: ../templates/component-template.yml
-  parameters:
-    PublishPackageArtifacts: # parameter is set in ADO
 
 variables:
   BuildConfiguration: 'Release'
   BuildPlatform: anycpu
   ComponentType: codeExtension # codeExtension, declarativeAsset, or generator
-  Language: dotnet
```

Comment:
I think you still want the Language variable

Reply:
Yep, this was a hack to publish when the eng was a bit less baked, we should add the language back

Reply:
Added back.